### PR TITLE
Add simple mergedata test for merging plain JSON strings

### PR DESCRIPTION
**Pull request by nickanderson**
In revewing other tests I saw cases for testing wrapping and merging with data containers, but I didn't see the simple case of just some plain json
as strings.

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a pull request! Maybe @vpodzime can review this?

File: tests/acceptance/01_vars/02_functions/mergedata-json-strings.cf [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/var/cfengine/bin/cf-agent -f-
+body common control
+{
+        inputs => {
+                    "../../default.cf.sub",
+                    "../../run_with_server.cf.sub"
```
Comment (vpodzime) [inline tests/acceptance/01_vars/02_functions/mergedata-json-strings.cf]: Why `run_with_server.cf.sub`?

**Comment by nickanderson** [comment]
I had an issue in the test, its passing now.

**Comment by vpodzime** [comment]
> I had an issue in the test, its passing now.

@cf-bottom let jenkins verify this

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2305/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/2305/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-2305/

**Pull request merged**